pkg/cmd/marathon/app/app_add.go

```diff
@@ -11,8 +12,19 @@ func newCmdMarathonAppAdd(ctx api.Context) *cobra.Command {
 		Use:   "add",
 		Short: "Add an application.",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return python.InvokePythonCLI(ctx)
+			client, err := marathon.NewClient(ctx)
+			if err != nil {
+				return err
+			}
+			if len(args) > 0 {
```

Comment:
I would use `Args:  cobra.ExactArgs(1),` (like in `job_remove.go`) instead of having this check to use the default cobra args handling.

Reply:
I don't know how that would work since the argument can just be empty and then the JSON is read from stdin.

Reply:
There's also a maximum number of args function. `cobra.MaximumNArgs(n)`?